src/persistence.interfaces.ts

```diff
@@ -85,12 +116,14 @@ export interface IPersistence {
     ): void;
     saveUserAttributesToPersistence(
         mpid: MPID,
-        userAttributes: UserAttributes
+        // TODO: UserAttributes is not a thing, but we have `AllUserAttributes`
+        //       Is there a difference?
```

Comment:
Confirming I think this can now be a TODO: go.mparticle.com/work/.... after what we discussed.  Yes `UserAttributes` is not a thing, but it shoudl be, and `AllUserAttributes` should be renamed `UserAttributes`.  I think perhaps not changing this until the next major version is better given the implications for definitely typed.  Best way to do this would be to create `UserAttributes` in definitely typed alongside `AllUserAttributes`, and have that exist until we go to the next major version, perhaps with a comment above it for now.